### PR TITLE
@W-9971168@: Added coverage enforcement in both nyc and gradle. Cover…

### PR DESCRIPTION
**Pull request by jfeingold35**
…age thresholds are tentative, and will likely increase as our coverage increases.

### CODE REVIEW

File: package.json
Comment (jfeingold35) [inline package.json:109]: I'm using `80` instead of `90` here, because the branch coverage is around 84%.

File: pmd-cataloger/build.gradle.kts
Comment (jfeingold35) [inline pmd-cataloger/build.gradle.kts:94]: I'm enforcing a minimum of 80% instead of 90%, because the coverage is exactly 80% at this time.

File: pmd-cataloger/build.gradle.kts
Comment (jbartolotta-sfdc) [inline pmd-cataloger/build.gradle.kts:4]: Let's set this to a specific version.

Comment (jfeingold35) [reply pmd-cataloger/build.gradle.kts:4]: @jbartolotta-sfdc , done. I've set it to 0.8.7.

**Review by jbartolotta-sfdc — CHANGES_REQUESTED** [review]
Let's wait for 3pp approval. I'm setting this to "Request Changes" so that we don't accidentally merge it.

**Review by jbartolotta-sfdc — APPROVED** [review]

**Pull request merged**
